Dockerfile.bindmount

```diff
@@ -0,0 +1,41 @@
+# 
+# This Dockerfile can be useful on macOS (perhaps Windows too), where the
+# the ./tools/build.sh script fails to run because of doxx memory issues:
```

Comment:
"the the" :wink:

Reply:
Well, it's not my fault that you didn't review it in time. There were some good 15 minutes between the PR was created and merged! LOL :-D